contracts/EvitaDAOToken.sol

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity ^0.8.0;
```

Comment:
we don't need this. Version is set up in hardhat config. Also pls read https://docs.soliditylang.org/en/develop/layout-of-source-files.html#version-pragma